docs/xks/developer-guide/scheduling-scaling.md

```diff
@@ -0,0 +1,280 @@
+                  operator: In
+                  values:
+                    - xks
+            topologyKey: topology.kubernetes.io/zone
+          weight: 100
+```
+
+This is an example configuration of podAntiAffinity for Prometheus. Spreading the pod deployment based on `topology.kubernetes.io/zone` and `topology.kubernetes.io/hostname` to only allow 1 pod on each node and to mitigate downtime in case an entire zone goes down, e.g: if a pod runs in zone A with key `prometheus` and value `xks` do not schedule it in zone A - choose zone B or C. Note that these settings are "preferred" and not required.
+
+We recommend using this configuration, as critical services should be distributed to multiple zones to minimize downtime.
+
+You can read more about this [in the official documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).
+
+## Pod Disruption Budget
+
+[Pod Disruption Budgets](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) are critical for any production deployment of an application. It enforces so that there are always a set amount of replicas of an application running. There is a risk that an application will during a short period of time have zero replicas
+running without if a Pod Disruption Budget has not been defined. XKS depends heavily on the existence of Pod Disruption Budgets to make sure that a cluster node pool can be scaled safely and upgrades can be applied to node pools without involving individual developers. During these types of event multiple Nodes will be drained.
+The Node will block any new Pods from being scheduled to it and start evicting all existing Pods running the Node during a drain. Without a Pod Disruption Budget all Pods belonging to the same Deployment may be stopped at the same time, before any new Pods have had the time to start. With a Pod Disruption Budget a limited
+amount of Pods will be stopped, and then started on a new Node. Eviction will continue with the remaining Pods after the new Pods are running and passed their readiness probe. This documentation is only relevant for applications that are deployed with multiple replicas. It is not possible to create a Pod Disruption Budget for
+a single replica application, one has to assume that downtime will most likely happen and an application is deployed as a single replica.
+
+Creating a Pod Disruption Budget can be very simple. Assume a Deployment named podinfo with the label `app: podinfo` and a replica count of three exists in a cluster. The Pod Disruption Budget below would make sure that at least two of the replicas will be running at all times. It is important that `minAvailible` is always
+smaller than `replicas`. It would be impossible to evict a Pod if the two values were equal. Removing a Pod would go against the Pod Disruption Budget, and creating an extra Pod would go against the replica count. This result is that a Node will never be able to evict Pods safely.

```

Comment:
"This result is that a Node will never be able to evict Pods safely"

I would highlight **never** to try to drive home this.